src/lib/openapi/spec/client-metrics-schema.ts

```diff
@@ -0,0 +1,61 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { dateSchema } from './date-schema';
+
+export const clientMetricsSchema = {
```

Comment:
Should make tests for this schema. Would be great to test some snapshots of data sent from the SDKs.